<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPersonalGoal</class>
 <widget class="QMainWindow" name="EditPersonalGoal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактировать личные цели</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Дата и время:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Цель:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="goal_lineEdit">
        <property name="placeholderText">
         <string>Введите цель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Фотографии:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="photo_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="photo_pushButton">
        <property name="text">
         <string>Прикрепить фотографии...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="image_label">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Не удалось открыть фотографию. Возможно файл поврежден.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="backPhoto_pushButton">
          <property name="text">
           <string>Предыдущая фотография</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextPhoto_pushButton">
          <property name="text">
           <string>Следующая фотография</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePhoto_pushButton">
          <property name="text">
           <string>Удалить фотографию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="save_pushButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
